lib/controllers/cars.js

```diff
@@ -0,0 +1,14 @@
+const { Router } = require('express');
+const CarModel = require('../models/Cars.js');
+
+// must update this file to use your database and models instead of hard coded cars data
+module.exports = Router()
+  .get('/', async (req, res) => {
+    const cars = await CarModel.getAll();
+    //   const filtered = cars.map(({ id, name }) => ({ id, name }));
+    res.json(cars);
+  })
+  .get('/:id', async (req, res) => {
```

Comment:
/:id should be first not / and go from most specific to least specific URL in the controler.